resources/lang/en/admin/hardware/form.php

```diff
@@ -8,6 +8,7 @@
   'bulk_delete_warn'	=> 'You are about to delete :asset_count assets.',
   'bulk_restore_warn'	=> 'You are about to restore :asset_count assets.',
     'bulk_update'		=> 'Bulk Update Assets',
+    'bulk_checkin'      => 'Bulk Checkin Assets',
```

Comment:
We can probably re-use the "Checkin Assets" or "Checkin" string here - it's a bulk menu, so I think the bulk part is implied.